tests/models/opt/test_modeling_tf_opt.py

```diff
@@ -281,14 +281,14 @@ def test_inference_no_head(self):
 
 
 @require_tf
-@slow
+# @slow
```

Comment:
Will need to be uncommented before merging.

Reply:
Think it's already fixed in the new commit